dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletRequest.java

```diff
@@ -2873,17 +2871,16 @@ public void pushParameterStack()
         if (WCCustomProperties.CHECK_REQUEST_OBJECT_IN_USE){
             checkRequestObjectInUse();
         }
-        SRTServletRequestThreadData reqData = SRTServletRequestThreadData.getInstance();
-        if (reqData.getParameters() == null)
+        if (threadData.getParameters() == null)
         {
-            reqData.pushParameterStack(null);
+            threadData.pushParameterStack(null);
```

Comment:
As you did previously this code can be removed and the push and pop stack methods can be removed from ThreadData.

Reply:
done